en/mulesoft/modules/ROOT/pages/ms_composer_sage_intacct_reference.adoc

```diff
@@ -0,0 +1,225 @@
+* *Attachment Name*
+* *Folder*
+* When adding files to an attachment, use a list as the source of the files, and specify a value in the following fields:
+** *File Binary Data*
+** *File Extension*
+** *File Name*
+
+Update Attachment::
+
+Updates an Attachment record. When updating an attachment, you can update the attachment properties or add new files from a list. Provide a value for *Attachment ID*.
+
+Get Attachments::
+
+Returns a list of Attachment records. To filter the list, provide a value for *Attachment ID*, *Attachment Name*, or *Folder Name*. To return the entire list, leave the fields blank.
+
+Create Attachment Folder::
+
+Creates a new folder for attachments. Provide a value for *Name*.
+
+Get Attachment Folders::
+
+Returns a list of attachment folders. To filter the list, provide a value for *Folder Name* to filter. To return the entire list, leave the fields blank.
+
+Create Customer::
+
+Create a new Customer record. Provide a value for the following:
+
+* *Customer ID*
+* *Primary Contact*
+* *Print As*
+
+Update Customer::
+
+Update a Customer record. Provide a value for *ID*.
+
+Get Customer::
+
+Returns a list of Customer records. To filter the list, provide a value for *Customer ID* or *Customer Name*. To return the entire list, leave the fields blank.
+
+Delete Customer::
+
+Deletes a Customer record. Provide a value for *Customer ID*.
+
+Create Order::
+
+Creates a new Order record and returns a value for the *Key* field. Provide a value for the following:
+
+* *Order Type*
+* *Date Created*
+* *Transaction Currency*
+* *Base Currency*
+* *Exchange Rate Type*
+* In the *Entries* > *Sotransitem* section, provide a value in the following:
+  ** *Sotransitem List*
+  ** *Item ID*
+  ** *Quantity*
+  ** *Location ID* +
+Note that the *Location ID* field is determined by the numbers preceding the location description. For example, if the location ID is `100-United States` then enter `100` in the *Location ID* field.
+
+  ** *Price*
+  ** *Unit*
+  ** If your order contains inventory type items or a mix of inventory and non-inventory items, provide a value for *Warehouse ID*.
+
+Update Order::
+
+Updates an Order record. +
+Note that when updating or reordering line items for an existing order, the line order can change. For example, if you drag a line item from the number two position to the number three position in the Sage Intacct UI, the line order changes. To avoid synchronizing the wrong Sage Intacct line order item, you must account for possible order line item changes. Prior to executing the *Update Order* action, identify the current *Line Number* and store it in a custom field within Sage Intacct. To delete a line item from an existing order, use the *Delete Order Line Item* action. +
```

Comment:
Can you split this into two parts? It is long, especially for text within a bullet.